src/cms/src/api/guideline-type/content-types/guideline-type/schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "kind": "collectionType",
+  "collectionName": "guideline_types",
+  "info": {
+    "singularName": "guideline-type",
+    "pluralName": "guideline-types",
+    "displayName": "Guideline-type",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "attributes": {
+    "guidelineNumber": {
+      "type": "integer",
+      "min": 1,
+      "default": 1,
+      "required": true,
+      "unique": true
+    },
+    "guidelineName": {
+      "type": "string",
+      "required": true
+    },
+    "icon": {
+      "type": "enumeration",
+      "enum": [
+        "list-check",
+        "person-hiking",
+        "car",
+        "dog-leashed",
+        "ban"
+      ],
+      "required": true
+    },
+    "defaultTitle": {
+      "type": "string",
+      "required": true
+    },
+    "defaultDescription": {
+      "type": "customField",
+      "options": {
+        "preset": "toolbar"
+      },
+      "customField": "plugin::ckeditor5.CKEditor",
+      "required": true
+    }
```

Comment:
Can we include a "defaultOrder" too?  If the order on the park-guideline isn't set then it would use the defaultOrder.  

We use the names "rank" and "precedence" on some other sorting fields.  But I'm okay with introducing a new field name in this case because I can never tell from these other labels if they are supposed to be ascending or descending.

Reply:
Change `order` to `rank`, also added `defaultRank` to `guidelineType`